### DISCUSSION TITLE
A javascript error occurred in the main process

### DISCUSSION THREAD

**Discussion by alshahabrezvi** [daltonmenezes/electron-router-dom#17] [Q&A]
when i  am trying to open my electron-vite app it show "A javascript error occurred in the main process". but in development process when i run ''electron-vite preview" it has no error. but after build it show error. when i uninstall electron router dom my error was gone. but i have to navigate to defferent page.. it anyone know how to fix please help me....

**Comment by daltonmenezes** [comment]
Hi @alshahabrezvi 

I just tested my [electron boilerplate](https://github.com/daltonmenezes/electron-app) and everything is working correctly, even updating the dependencies to their latest versions! It's really very difficult to see exactly what your error is without having your code to run here! I always ask people to create a test repository that reproduces the error they are having, if you can do this I will be able to help you in a better way!
